source/commands.txt

```diff
@@ -0,0 +1,176 @@
+.. _vsce-commands:
+
+============================
+MongoDB for VS Code Commands
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This document lists the VS Code commands available for
+|vsce|. To open the Command Palette and execute these commands:
+
+.. include:: /includes/list-tables/command-palette-options.rst
+
+.. include:: /includes/admonitions/tip-mongodb-command-palette.rst
```

Comment:
Nit: suggest including that 'in development' admonition from the install page here.

Reply:
Oh oops thought I had it. Thanks for flagging